frontend/src/components/SidebarHint.tsx

```diff
@@ -29,7 +29,11 @@ function SidebarHint() {
         title="New Channel"
         description="You're currently opening a new channel"
         buttonText="View Channel"
-        buttonLink="/channels/order"
+        buttonLink={`${
```

Comment:
how does this work?

Reply:
how does the outgoing flow work?

And when I click this link when I'm currently opening an incoming channel, what I see on the page is wrong.

Reply:
change back to "channels/order"